src/bp-activity/bp-activity-functions.php

```diff
@@ -1253,7 +1253,22 @@ function ( $carry, $user_id ) use ( $current_user_id, $like_text ) {
 function bp_activity_favorites_upgrade_data() {
 	$bp_activity_favorites = bp_get_option( 'bp_activity_favorites', false );
 
-	if ( ! $bp_activity_favorites ) {
+	if ( ! $bp_activity_favorites && bp_is_active( 'activity' ) ) {
+
+		if ( bp_is_large_install( 100 ) ) {
```

Comment:
@chetansatasiya why do we need to check only 100 users as large network? is it neccessary. 100 users is not large network.

Reply:
@mehulkaklotar made the changes.

Reply:
@mehulkaklotar At this moment we have a condition like `10000` users do you think on `init` action it will handle `9000` members will handle in single request?

Reply:
@mehulkaklotar If we processed via `10000` users I am sure like in many servers `mysql` service will go out of service and crashed and platform will not going to activate.

Reply:
@chetansatasiya no it won't happen. because the query is only to count not fetching the rows. I have tested in my local with minimal config server with more than 10k users and count query doesn't even take more than a half second to execute.